TypeCobol.Test/Parser/EILegacy/EITestCpyList2A.rdzPGM.txt

```diff
@@ -0,0 +1,15 @@
+--- Diagnostics ---
+Line 1[8,23] <37, Warning, General> - Warning: "END PROGRAM" is missing.
```

Comment:
We should not have this diagnostic but as discussed with @fm-117 it seems it goes beyond the scope of this issue so we'll see about this another time.